explore-analyze/visualize/charts/area-charts.md

```diff
@@ -0,0 +1,168 @@
+---
+navigation_title: Area charts
+applies_to:
+  stack: ga
+  serverless: ga
+description: Instructions and best practices for building area charts with Kibana Lens in Elastic.
+---
+
+# Build area charts with {{kib}}
+
+Area charts are line charts with the area below the line filled in with a certain color or texture. Area charts work with numeric metrics over the horizontal axis (typically time) and are ideal to display quantitative values over an interval or time period, to show trends for time series like traffic, CPU, revenue, or error rates.
+
+You can create area charts in {{kib}} using [**Lens**](../lens.md).
+
+![Example Lens area chart](../../images/kibana-area-chart.png)
+
+## Build an area chart
+
+To build an area chart:
+
+::::::{stepper}
+
+:::::{step} Access Lens
+**Lens** is {{kib}}'s main visualization editor. You can access it:
+- From a dashboard: On the **Dashboards** page, open or create the dashboard where you want to add an area chart, then add a new visualization.
+- From the **Visualize library** page by creating a new visualization.
+:::::
+
+:::::{step} Set the visualization to Area
+New visualizations often start as **Bar** charts.
+
+Using the **Visualization type** dropdown, select **Area**.
+:::::
+
+:::::{step} Define the data to show
+1. Select the {{data-source}} that contains your data.
+
+2. Drag a time-based field to the **Horizontal axis** and numeric field to the **Vertical axis**. You can use aggregation functions like `Date histograms` and `Filters`, or create custom calculations with [formulas](../lens.md#lens-formulas).
+
+Optionally, you can add more numeric fields to create additional series, or drag a categorical field to **Break down** to split the series.
```

Comment:
What does this mean? Do we expect users to know what a new series is or what it means to break down a series?